ert_shared/ensemble_evaluator/entity/function_step.py

```diff
@@ -67,7 +66,7 @@ def run_job(self, job, transmitters: Dict[str, "RecordTransmitter"], client):
             )
         return output
 
-    def run(self, inputs: Dict[str, "RecordTransmitter"]):  # type: ignore
+    def run(self, inputs: Dict[str, RecordTransmitter]):  # type: ignore
```

Comment:
Is it still needed to ignore typing here?

Reply:
Yes, the ignore was not for the RecordTransmitter, it is because the run functions overrides a prefect functions that does weird stuff that mypy cannot handle.